website/www/site/content/en/documentation/dsls/sql/extensions/create-external-table.md

```diff
@@ -204,6 +205,131 @@ TYPE bigquery
 LOCATION 'testing-integration:apache.users'
 ```
 
+## Cloud Bigtable
+
+### Syntax
+
+```
+CREATE EXTERNAL TABLE [ IF NOT EXISTS ] tableName (
+    key VARCHAR NOT NULL,
+    family ROW<qualifier cells [, qualifier cells ]* >
+    [, family ROW< qualifier cells [, qualifier cells ]* > ]*
+)
+TYPE bigtable
+LOCATION 'googleapis.com/bigtable/projects/[PROJECT_ID]/instances/[INSTANCE_ID]/tables/[TABLE]'
+```
+
+*   `key`: key of the Bigtable row
+*   `family`: name of the column family
+*   `qualifier`: the column qualifier
+*   `cells`: Either of each value:
+    *   `TYPE`
+    *   `ARRAY<SIMPLE_TYPE>`
+*   `LOCATION`:
+    *   `PROJECT_ID`: ID of the Google Cloud Project.
+    *   `INSTANCE_ID`: Bigtable instance ID.
+    *   `TABLE`: Bigtable Table ID.
+*   `TYPE`: `SIMPLE_TYPE` or `CELL_ROW`
+*   `CELL_ROW`: `ROW<val SIMPLE_TYPE [, timestampMicros BIGINT [NOT NULL]] [, labels ARRAY<VARCHAR> [NOT NULL]]`
+*   `SIMPLE_TYPE`: on of the following:
+    *   `BINARY`
+    *   `VARCHAR`
+    *   `BIGINT`
+    *   `DOUBLE`
+    *   `BOOLEAN`
```

Comment:
So what is the reason that single type only supports these 5 types (i.e. not include other SQL types like TIMESTAMP)?

Reply:
BigQuery supports only those 5 types. But I agree that there is no reason why not to support others as well. Should I do that in this PR or in the next one?

Reply:
Maybe just do that in this PR to make this feature complete?

Reply:
Ok, will do!